src/controllers/messagesController.ts

```diff
@@ -226,7 +226,7 @@ export default class MessagesController {
     | IResponseSuccessOctet<Buffer>
   > =>
     withUserFromRequest(req, user =>
-      withGetThirdPartyAttachmentParams(req, (messageId, attachmentUrl) =>
+      withGetThirdPartyAttachmentParams(req, async (messageId, attachmentUrl) =>
```

Comment:
why async?

Reply:
It was a remnant from a refactor. Fixed in https://github.com/pagopa/io-backend/pull/966/commits/5e3db4f3f70cfa3dc8127cbc83933fddbfa27a65